operations/mimir-mixin/alerts/alerts.libsonnet

```diff
@@ -127,7 +127,7 @@
         {
           alert: $.alertName('FrontendQueriesStuck'),
           expr: |||
-            sum by (%s) (cortex_query_frontend_queue_length) > 1
+            sum by (%s) (cortex_query_frontend_queue_length{container="query-frontend"}) > 1
```

Comment:
This wouldn't work when deployed in monolithic mode. Why do you need to distinguish between deployments? I think a generic alert would be enough.

If you want the alert to be more expressive, you could also group by `job`, and then include the `job` in the alert message.